README.md

```diff
@@ -37,6 +37,11 @@ DatabaseEngine methods `#getPSResultSet(final String name)`, `#getPSIterator(fin
 (final String name, final int fetchSize)`, since 2.7.0, throw an extra `ConnectionResetException` when the database
 connection is lost and recovered.
 
+H2 2.x is not backward compatible with H2 1.x. Two changes worth some comments:
```

Comment:
I think this will require a release of PDB 2.9.0. So please put this text in a new header (## Changes from 2.9.0) after ## Changes from 2.8.0

Reply:
:+1: